SwiftGen.playground/Pages/Storyboard-Demo.xcplaygroundpage/Contents.swift

```diff
@@ -86,4 +101,12 @@ validateVC.title
    not known by the storyboard. But it should work correctly in a real project. */
 // let cgu = UIStoryboard.Name.Wizzard.createAccountViewController
 
+let segue = UIStoryboard.Segue.Message(rawValue: "CustomSegue")!
```

Comment:
Could be a good idea to add an example with real-world code BTW, like how you would use that enum to typically implement `prepareForSegue` (even in case you need to comment that code if it doesn't work in a Playground context, that'd still be useful for the reader that opens the playground to understand typical usages)